MacaoProject/Macao/Plyaer.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Macao
+{
+    public class Player
+    {
+        public string fisrtName { get; set; }
+        public string lastName { get; set; }
+
+        public void LastCard()
```

Comment:
the name of the method does not express an action, like Pick or Put try to find another name